### PR TITLE
Add support for Romania 

### LINKED ISSUE #5974: Add support for Romania
Add support for Romania as part of the new international expansion project.

The following is a checklist to help with this:
- [x] Add the country to `supported_countries` function
- [x] Add the country to `countryFeeStripeDocsSectionNumbers` function
- [ ] Onboard on Stripe using a test account form the country (make sure to share it in PR for testers)
- [ ] Make a payment using a test card
- [ ] Make sure the currency is showing properly
- [ ] Turn on multiple payment methods and make a test payment with one
- [ ] Simulate a dispute using one of the [test cards](https://stripe.com/docs/testing?testing-method=card-numbers#disputes). Make sure the disputes are working properly (ie, showing in the list + currency is good)
- [ ] Make sure the deposits are working properly (ie, double check with Stripe Docs the minimum deposits amount, frequency, wait time...)

### PR DESCRIPTION
**Pull request by dpaun1985**
Fixes #5974 

#### Changes proposed in this Pull Request

Add RON currency support.

#### Testing instructions

* On server side checkout 3264-gh-Automattic/woocommerce-payments-server branch

* **Base fees**
* Go to Woo -> Settings -> General and set Store Country to Romania and currency to Romanian Leu
* Remove supported countries from the account on server side
* Place an order - base fees should be applied
Screenshot 2023-04-18 at 18 16 54

* **FX fees**
* change store currency to EUR
* Remove supported countries from the account on server side
* Place an order - base and FX fees should be applied
Screenshot 2023-04-18 at 18 19 45

* **EPS fees**
* Place an order with EPS 
* You will be redirected to a Stripe test page where you can approve/fail the order
* Approve the order - EPS fees should be applied
Screenshot 2023-04-18 at 18 29 39

Continue with all fees defined for Romania.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6063` or branch name `add/romania-fees` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/romania-fees) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 08a8688b395a3a394777526b7811528d4f8fa3d9
- Build time: 2023-05-04 09:07:26 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/add-romania-fees [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: add
+
+Add currency fees for RON (The currency of Romania)
```
Comment (anu-rock) [inline changelog/add-romania-fees]: IMO, it doesn't reflect the actual scope of changes. There are no currency-related changes in this PR. The title of #6088 is I think more appropriate here.

**Review by anu-rock — APPROVED** [review]
LGTM 👍

File: changelog/add-romania-fees (hunk 1 above)
Comment (dpaun1985) [reply changelog/add-romania-fees]: changed. thx.

**Review by mordeth — APPROVED** [review]
LGTM! ✅

**Comment by elazzabi** [comment]
Removed the changelog in https://github.com/Automattic/woocommerce-payments/pull/6063/commits/0de0949c7c93e9c9913b8ae46a808fcada462738 given we'll have one changelog for the 6 countries

**Pull request merged**